src/pages/boards/[id].tsx

```diff
@@ -39,8 +39,8 @@ const Board = () => {
   return (
     <>
       <BoardImage
-        src={drelloBoardsList[2].boardImage?.src || "/images/template-1.JPG"}
-        alt={drelloBoardsList[2].boardImage?.alt}
+        src={drelloBoardsList[0].boardImage?.src || "/images/template-1.JPG"}
```

Comment:
Can we use a constant for image paths for maintainability?

```suggestion
        src={drelloBoardsList[0].boardImage?.src || template1 }
```

e.g.
```typescript
// src/constants/image-path.ts
export const template1 = "/images/template-1.JPG"
```